doc/man5/config.pod

```diff
@@ -42,8 +42,8 @@ If B<pathname> is a simple filename, that file is included directly at
 that point.  Included files can have B<.include> statements that specify
 other files.  If B<pathname> is a directory, all files within that directory
 that have a C<.cnf> or C<.conf> extension will be included.  (This is only
-available on systems with POSIX IO support.)  Multiple levels of directory
-inclusion are not supported.
+available on systems with POSIX IO support.)  Any sub-directories found
+inside the B<pathname> are B<ignored>.
```

Comment:
Not just that, but also further .include directives inside the included files will work only if including files and not directories. In other words, recursively including directories in directory includes is not supported. Maybe this should be moved to the BUGS section as that is a little bit arbitrary limitation.

Reply:
added a BUGS entry, tried to fix this wording.